.github/test_env_dockerfile

```diff
@@ -1,33 +1,30 @@
 # start from Earthwave's agreed common python starting image
-FROM continuumio/miniconda3:4.11.0
+FROM continuumio/miniconda3
```

Comment:
I haven't seen this line in the other ones, are we just grabbing the latest image here?

Reply:
That's correct. The python template upgrade I just completed has, as one intention, making the CI/CD chain "evergreen", i.e. constantly renewing itself.

There will be more work to do to change _python_ versions at some point, but I've managed to render the versions of almost everything else as "latest".

The reason you haven't seen this line in the over template upgrade PRs is because it's now located [here](https://github.com/earthwave/ew_continuous_integration/blob/main/ci_base_containers_dockerfile) for almost all Earthwave repositories. Glambie is an exception since, being open source, it cannot access our internal package management systems.

Reply:
Makes sense, thanks for explaining!